oap-stdlib/src/test/resources/oap/application/KernelProfileTest/module6.conf

```diff
@@ -24,7 +24,7 @@ services {
   }
   profile2 {
     name = profile
-    profile = -run
+    profile = disable
```

Comment:
What profile is disabled here? Earlier it was `run`, but now I can't see diabled profile name. Could you clarify please?

Reply:
With this property I disable profile1 which has the same name "profile".